cmd/inspector/main.go

```diff
@@ -445,6 +463,62 @@ func CreateCSVStats(cmd *cobra.Command, args []string) (err error) {
 	return nil
 }
 
+// ObjectHealth gets information about the health of an object on the network
+func ObjectHealth(cmd *cobra.Command, args []string) (err error) {
+	ctx := context.Background()
+
+	i, err := NewInspector(*Addr, *IdentityPath)
+	if err != nil {
+		return ErrArgs.Wrap(err)
+	}
+
+	req := &pb.ObjectHealthRequest{
+		ProjectId:         []byte(args[0]),
+		Bucket:            []byte(args[1]),
+		EncryptedPath:     []byte(args[2]),
+		StartAfterSegment: 0,
+		EndBeforeSegment:  0,
+		Limit:             0,
+	}
+
+	resp, err := i.healthclient.ObjectHealth(ctx, req)
+	if err != nil {
+		return ErrRequest.Wrap(err)
+	}
+
+	fmt.Printf("%+v\n", resp)
+
+	return nil
+}
+
+// SegmentHealth gets information about the health of a segment on the network
+func SegmentHealth(cmd *cobra.Command, args []string) (err error) {
+	ctx := context.Background()
+
+	i, err := NewInspector(*Addr, *IdentityPath)
+	if err != nil {
+		return ErrArgs.Wrap(err)
+	}
+
+	segmentIndex, err := strconv.ParseInt(args[1], 10, 64)
+
+	req := &pb.SegmentHealthRequest{
+		ProjectId:     []byte(args[0]),
+		SegmentIndex:  segmentIndex,
+		Bucket:        []byte(args[2]),
+		EncryptedPath: []byte(args[3]),
+	}
+
+	resp, err := i.healthclient.SegmentHealth(ctx, req)
+	if err != nil {
+		return ErrRequest.Wrap(err)
+	}
+
+	fmt.Printf("%+v\n", resp)
```

Comment:
I see your point to @aligeti but seems sloppy. Why not just return properly here?